app/actions/legalAgreement.ts

```diff
@@ -42,7 +42,20 @@ export const loadAgreements: ThunkAction<{}, IAppState, {}> = async (dispatcher,
   const [reservationAgreement, tokenHolderAgreement] = await Promise.all([
     getDocumentFromIPFS(agreementHashes.reservationAgreementHash),
     getDocumentFromIPFS(agreementHashes.tokenHolderAgreementHash),
-  ]);
+  ]).catch((...errors: Error[]) => errors.map(console.error));
```

Comment:
here because the IPFS link was broken the errors were not being caught and it would crash